src/nnet3/nnet-chain-training2.h

```diff
@@ -0,0 +1,192 @@
+// nnet3/nnet-chain-training.h
+
+// Copyright    2015  Johns Hopkins University (author: Daniel Povey)
+//                2019    Idiap Research Institute (author: Srikanth Madikeri)
+
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef KALDI_NNET3_NNET_CHAIN_TRAINING_H_
```

Comment:
please fix include guard.  There is a script in misc/ that is for that, you might find other bad ones.

Reply:
I modified it manually. When I ran misc/maintenance/fix_include_guards.sh, it changed a whole bunch of files not related to this PR.